### PR TITLE
#181339492: User-role-settings

### PR DESCRIPTION
**Pull request by blandri**
#### What does this PR do?

  Intergrate user settings to the app

#### Description of Task to be completed?

- Super admin should be able to assign or update different roles to the users.

#### How should this be manually tested?

- Go to this deployed preview [preview](https://deploy-preview-19--barefoot-nomad-fe.netlify.app/)
- Log in as super admin
- Go to settings
- Assign a role

#### What are the relevant pivotal tracker stories?

- [#181339492](https://www.pivotaltracker.com/n/projects/2555280)

#### Screenshots (if appropriate)
image

### CODE REVIEW

File: src/redux/actions/user_role_settings.action.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import {toast} from 'react-toastify'
+
+export const assignRoleAction=(payload)=>{
+  return {
+    type: ASSIGNROLE,
+    payload
+}
+}
+
+// export const assignRoleActionError=(payload)=>{
```
Comment (Tsa2341) [inline src/redux/actions/user_role_settings.action.js:12]: remove commented lines, do it in all the others files also,

File: src/redux/actions/user_role_settings.action.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+export const getUsers = () => async (dispatch) => {
+  
+  try {
+    await dispatch({
+      type: GETUSERS,
+    });
+    const res = await axiosInstance(`/users`, { method: 'GET' });
+    let users=res.data.data.map(user=>user.email)
+    console.log(users);
```
Comment (Tsa2341) [inline src/redux/actions/user_role_settings.action.js:57]: remove unnecesssary console.logs, do it in all the others files also,

File: tests/user_role_settings.test.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  
+    stor.dispatch(assignRoleAction(mes))
+    await waitFor(() => {
+      stor.subscribe(() => {
+        expect(stor.getState().assignRoleReducer).toEqual(expectation);
+      });
+    });
+  });
+
+//   it('should test get all roles action',async()=>{
```
Comment (Tsa2341) [inline tests/user_role_settings.test.js:64]: remove commented lines, do it in all the others files also,

File: src/components/user_role.js [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+    <FormControl sx={FormControlSX}>
+            <InputLabel id="demo-simple-select-label">Role</InputLabel>
+            <Select
+              
+              value={role}
+              /* istanbul ignore next */
+              onChange={(e) => setRole(e.target.value)}
+              
+              label="Role"
+              // sx={{
```
Comment (Tsa2341) [inline src/components/user_role.js:155]: remove commented lines, do it in all the others files also,

**Review by Tsa2341 — CHANGES_REQUESTED** [review]
Remove the scrolling on the page i think you it can fit and center without the scrolling.

**Review by Gabin-ishimwe — APPROVED** [review]

**Review by Code-250 — APPROVED** [review]

**Review by Eli250 — APPROVED** [review]

**Pull request merged**